dags/vz_cr3_extract_ocr_narrative.py

```diff
@@ -29,35 +29,39 @@
     "HASURA_ADMIN_KEY": {
         "opitem": "Vision Zero CRIS Import",
         "opfield": "production.GraphQL Endpoint key",
-    }, 
+    },
     "AWS_ACCESS_KEY_ID": {
         "opitem": "Vision Zero CRIS Import",
         "opfield": "production.AWS Access key",
     },
     "AWS_SECRET_ACCESS_KEY": {
         "opitem": "Vision Zero CRIS Import",
         "opfield": "production.AWS Secret key",
-    }
+    },
 }
 
 with DAG(
     dag_id=f"vz_cr3_ocr_narrative_extract_{DEPLOYMENT_ENVIRONMENT}",
     default_args=default_args,
     # Every 5 minutes, at 8A, 9A, and 10A
-    schedule_interval="*/5 8-10 * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
+    schedule_interval="*/20 * * * *"
+    if DEPLOYMENT_ENVIRONMENT == "production"
+    else None,
```

Comment:
Adjust the schedule from 5-minutely for 3 hours per day (36 executions), to 20-minutely all day (72 executions).